README.md

```diff
@@ -94,156 +94,146 @@ Example Job
-                             text_keywords=['python'], 
-                             created_at_min='2020-07-09T13:35:11+0000')
-```
-source_keys is required
-
-* Profile parsing:   
-Add a profile resume as binary to a given source
-
+### 📖 **The Profile Object**
+JSON object with 5 required fields: `key`, `reference`, `info`, `text_language` and `text`. The JSON contains other optional fields that cannot be listed here. (See the examples above)
+### 🧠 **Parse a Resume in a Source**
+This endpoint allows you to parse a resume and make a profile object from it.
+> 📘 **Real-time parsing**: To use the real-time parsing feature, you must have it enabled for the correponding source. In which case you just need to set `sync_parsing` to `1`.
+- Open the file in `binary mode`
 ```python
 with open('path/2/file', "rb") as f:
     profile_file = f.read()
-
-resp = client.profile.parsing.add_file(source_key="source_key", 
+```
+- Parse it using this method without reference:
+```python
+response = client.profile.parsing.add_file(source_key="source_key", 
                                   profile_file=profile_file, sync_parsing=1,
                                   sync_parsing_indexing=1,
                                   webhook_parsing_sending=0, 
                                   tags=[{"name":"archive", "value":True}])
 ```
 
-* Retrieve Parsing Object:     
+- Or using a reference like this:
+```python
+response = client.profile.parsing.add_file(source_key="source_key", reference = "my_resume",
+                                  profile_file=profile_file, sync_parsing=1,
+                                  sync_parsing_indexing=1,
+                                  webhook_parsing_sending=0, 
+                                  tags=[{"name":"archive", "value":True}])
+```
+In both cases the output should look like this:
+```
+{'code': 201,
+ 'message': 'Profile parsed successfully. Profile extraction finished : 8.00 seconds.',
+ ...}
+ ```
+```python
+client.profile.searching.list(source_keys=["source_key"], page=1, limit=30,.
+                             sort_by='created_at', order_by="desc",
+                             text_keywords=['python'], 
+                             created_at_min='2020-07-09T13:35:11+0000')
+```
+### 🧠 **Get a Resume Parsing from a Source**  
 Retrieve Parsing information using source key and key/reference.
-
+> ⚠️ **Query parameters**: `reference` and `key` cannot be null at the same time.
+- Retrieve parsing information using source key
+```python 
+response = client.profile.parsing.get(source_key="source_key", key="profile_key")
+```
+- Retrieve parsing information using reference
 ```python
-resp = client.profile.parsing.get(source_key="source_key", key="profile_key")
+response = client.profile.parsing.get(source_key="source_key", reference="my_resume")
+```
+- The output should should look like this
+```
+{'code': 200,
+ 'message': 'Profile parsing',
+ ...}
 ```
```

Comment:
This is the async output --> a message indicating that your profile has been sent to the queue and will be parsed asynchronously. 

If you would like to receive the result directly in the response body --< try to use a real-time parsing source and set the parameter sync_parsing to 1 .

Notice that if you set it to 1 and the source is not a real time source --< by default it will  be asynchronous (ajoute un truc comme ça) https://developers.hrflow.ai/reference/parse-a-resume (real time parsing ctrl+c ctrl +v )

Reply:
C'est bon j'ai distingué les deux cas (sync et async) et j'ai mis un output example pour chacun des deux